src/network/message.rs

```diff
@@ -147,7 +148,14 @@ pub enum NetworkMessage {
     Ping(u64),
     /// `pong`
     Pong(u64),
-    // TODO: bloom filtering
+    /// `merkleblock`
+    MerkleBlock(MerkleBlock),
```

Comment:
Maybe add the bip number to the docstring like it was done for the BIP157 messages below?

Reply:
Done